### DISCUSSION TITLE
day2 로봇 구동 실습 중 질문 있습니다.

### DISCUSSION THREAD

**Discussion by khehaeun** [Road-Balance/gcamp_ros2_basic#46] [Q&A]
Day2_Node_Package_Launch.pdf
34페이지, 35페이지 진행 중
$ ros2 run teleop_twist_keyboard teleop_twist_keyboard --ros-args -r __ns:=/skidbot
를 입력했으나 초록색 로봇이 가제보에 나타나지 않았습니다.
또한 35페이지에서 rqt_graph를 입력하였으나 첨부한 사진처럼 강의자료와 동일한 그래프가 나타나지 않았습니다.
rqt_graph를 cd ~/ros2_ws/src/gcamp_ros2_basic 이 위치에서 입력하였는데 잘못된 방법인지.. 로봇이 가제보에 나오지 않는 원인은 무엇인지 여쭤보고자 합니다.
![2022-07-14 (2)](https://user-images.githubusercontent.com/108414567/178915962-c95c5bdc-97fa-47ae-b05e-027afe79a884.png)

또한 :~/ros2_ws/src/gcamp_ros2_basic$
이 위치에서 code . 을 입력하시면 vs code 프로그램이 열리 것을 보고 저 또한 vscode를 설치 후 code.를 입력하였으나 vscode 프로그램이 전혀 동작하지 않았습니다.
![2022-07-14 (4)](https://user-images.githubusercontent.com/108414567/178920964-3d68ccc8-4ab7-4848-a9e2-7f6b55c48fa5.png)

**Comment by kimsooyoung** [comment]
1. gazebo를 실행하셨던 터미널 로그를 보신 뒤, 혹시 discussion에 같은 문제를 겼었던 경우가 있었나 찾아보세요. 해당 문제는 많이 등장한 질문이라 충분한 정보를 얻으실 수 있을 것입니다.
2. vscode와 같은 개발자 도구는 별도 시간을 통해 알려드리겠습니다.

**Reply by kimsooyoung** [reply]
문제 해결 되셨을까요?
